APPIUM.md

```diff
@@ -0,0 +1,64 @@
+Cordova Medic Appium Runner
+===========================
+
+This document describes a method to run Cordova Appium tests without full-blown CI setup with [Camera][plugin_camera] and [Contacts][plugin_contacts] plugins taken as an example.
+Appium tests can be ran on Android and iOS.
+
+## Setup
+
+Clone medic repository by running
+
+``` shell
+ > git clone https://github.com/apache/cordova-medic
+```
+
+You will need an app to test, so create it and add the platform and the plugins:
+
+``` shell
+ > cordova create test-app
+ > cd test-app
+ > cordova platform add <android or ios>
+ > cordova plugin add cordova-plugin-camera cordova-plugin-contacts
+```
+
+### Running on Android
+
+Cordova Appium tests for Android can be ran on either real device or an emulator. If you're running tests on the emulator you will need to launch it manually before the test run. Please also note that Appium tests are written with Android API 19 (KitKat) and 21 (Lollipop) in mind. They're not yet tested on higher versions of Android OS.
```

Comment:
Are there any OS/software requirements to run Appium tests? Like min OSX version/Xcode version or something?